miso-web/src/main/webapp/pages/adminsub.jsp

```diff
@@ -50,6 +50,11 @@
         <li><a href="<c:url value="/miso/indices"/>">Indices</a></li>
         <li><a href="<c:url value="/miso/experiments"/>">Experiments</a></li>
         <li><a href="<c:url value="/miso/studies"/>">Studies</a></li>
+      <sec:authorize access="hasRole('ROLE_ADMIN')">
+        <br/>
+        <li><a href="<c:url value="/miso/admin/users"/>">Users</a></li>
+        <li><a href="<c:url value="/miso/admin/groups"/>">Groups</a></li>
```

Comment:
Users and Groups should probably still be under a separate 'User Administration' heading